waspc/packages/deploy/src/providers/fly/setup/setup.ts

```diff
@@ -85,6 +85,7 @@ async function setupServer(deploymentInfo: DeploymentInfo<SetupOptions>) {
 	await $`flyctl launch --no-deploy ${launchArgs}`;
 
 	const minMachinesOptionRegex = /min_machines_running = 0/g;
+	const internalPortOptionRegex = /internal_port = \d+/g;
```

Comment:
We already defined this regex in the `setupClient` fn, and since it's the same thing, I'd extract this regex to the top and use it both `setupServer` and `setupClient` functions.